scripts/load_testing.py

```
@@ -51,40 +53,33 @@ def __init__(self, thread_number):
 
             # Lock for PID
             UpdateThread.locks[pid].acquire()
 
-            url = f'https://{UpdateThread.host}/api/{UpdateThread.index}/{pid}'
+            url = f"https://{UpdateThread.host}/api/{UpdateThread.index}/{pid}"
 
             try:
-                response = requests.get(url,
-                                        verify=False,
-                                        headers=self.headers)
+                response = requests.get(url, verify=False, headers=self.headers, timeout=UpdateThread.timeout)
```

Ruff (0.15.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)